File: app/src/main/java/com/se/music/activity/MusicStateListener.java
package com.se.music.activity;

/**
 * Created by gaojin on 2017/3/22.
 * <p>
 * 音乐状态接口
 */

public interface MusicStateListener {
    /**
     * 更新歌曲状态信息
     */
    void updatePlayInfo();

    /**
     * 更新播放时间
     */
    void updateTime();

    void reloadAdapter();
}

File: app/src/main/java/com/se/music/activity/MusicStateListener.kt
package com.se.music.activity

/**
 *Author: gaojin
 *Time: 2018/5/13 下午5:44
 */

interface MusicStateListener {
    /**
     * 更新歌曲状态信息
     */
    fun updatePlayInfo()

    /**
     * 更新播放时间
     */
    fun updateTime()

    fun reloadAdapter()
}